### ISSUE TITLE
Drop torch dependency

### ISSUE THREAD

**Issue by jluethi** [jluethi/fractal-faim-hcs#1]
Currently including a torch dependency, because otherwise I run into:

```
/Users/joel/mambaforge/envs/fractal-faim-hcs-test/lib/python3.9/site-packages/anndata/experimental/pytorch/_annloader.py:15: in <module>
    import torch
E   ModuleNotFoundError: No module named 'torch'

During handling of the above exception, another exception occurred:
tests/test_fractal_tasks.py:10: in <module>
    from fractal_faim_hcs.fractal_create_ome_zarr_md import create_ome_zarr_md
src/fractal_faim_hcs/fractal_create_ome_zarr_md.py:10: in <module>
    from faim_hcs.Zarr import build_zarr_scaffold
/Users/joel/mambaforge/envs/fractal-faim-hcs-test/lib/python3.9/site-packages/faim_hcs/Zarr.py:11: in <module>
    from anndata.experimental import write_elem
/Users/joel/mambaforge/envs/fractal-faim-hcs-test/lib/python3.9/site-packages/anndata/experimental/__init__.py:2: in <module>
    from .pytorch import AnnLoader
/Users/joel/mambaforge/envs/fractal-faim-hcs-test/lib/python3.9/site-packages/anndata/experimental/pytorch/__init__.py:1: in <module>
    from ._annloader import AnnLoader
/Users/joel/mambaforge/envs/fractal-faim-hcs-test/lib/python3.9/site-packages/anndata/experimental/pytorch/_annloader.py:18: in <module>
    warnings.warn("Сould not load pytorch.")
E   UserWarning: Сould not load pytorch.
```

Not sure why this happens, the anndata code is supposed to handle that issue with a try except here: https://github.com/scverse/anndata/blob/e067515887e0badfa5e02b7a75ac66679215ec2c/anndata/experimental/pytorch/_annloader.py#L15

Test it further, see if I can drop the dependency

**Comment by jluethi** [comment]
Actually, even with anndata 0.8.0, we can import write_elem from the _io.specs subpackage already and it works. But importing from experimental leads to torch issues.

**Comment by jluethi** [comment]
Closed with [0c2103b03f661c80fee9ccbd27a2f3f26ded4ae4 ](https://github.com/jluethi/faim-hcs/commit/0c2103b03f661c80fee9ccbd27a2f3f26ded4ae4) in faim-hcs branch.
